src/coreclr/nativeaot/docs/compiling.md

```diff
@@ -2,21 +2,8 @@
 
 This document explains how to compile and publish your project using Native AOT toolchain. First, please _ensure that [pre-requisites](prerequisites.md) are installed_. If you are starting a new project, you may find the [HelloWorld sample](https://github.com/dotnet/samples/tree/main/core/nativeaot/HelloWorld/README.md) directions useful.
 
-## Add ILCompiler package reference
+Please consult [documentation](https://github.com/dotnet/docs/blob/main/docs/core/deploying/native-aot.md) for instructions how to compile application.
```

Comment:
```suggestion
Please consult [documentation](https://github.com/dotnet/docs/blob/main/docs/core/deploying/native-aot.md) for instructions how to compile application.

The instructions remaining in this document discuss advanced topics only.
```

Reply:
Should we also include a link to creating the [rd.xml](https://github.com/dotnet/runtime/blob/main/src/coreclr/nativeaot/docs/rd-xml-format.md) file here?

@agocke was thinking ways to unblock developers when they run into issues with native aot and not wait for the SDK to pick the fix. Perhaps, this doc can serve that purpose as well?

Reply:
We have several other documents on troubleshooting, including rd.xml. https://github.com/dotnet/runtime/tree/main/src/coreclr/nativeaot/docs